internal/common/db.go

```diff
@@ -20,8 +20,21 @@ const (
 	ProgressWeightPreparingForInstallationStage float64 = 0.1
 	ProgressWeightInstallingStage               float64 = 0.7
 	ProgressWeightFinalizingStage               float64 = 0.15
+
+	NotificationTypeCluster  = "ClusterState"
+	NotificationTypeEvent    = "Event"
+	NotificationTypeHost     = "HostState"
+	NotificationTypeInfraEnv = "InfraEnv"
 )
 
+type Notifiable interface {
+	GetClusterID() *strfmt.UUID
+	GetInfraEnvID() *strfmt.UUID
+	GetHostID() *strfmt.UUID
+	GetNotificationType() string
+	GetResource() any
```

Comment:
You can remove `Get` from all of these names.

Reply:
That's incredibly unfortunate, wow. I guess that's the price we pay for being able to use functions as values :tired_face: